system/services/admin/AuditService.cfc

```diff
@@ -32,6 +42,49 @@ component output="false" singleton=true {
 		} );
 	}
 
+	public query function getAuditLog(
+		  numeric startRow = 1
+		, numeric maxRows  = 0
+		) output=false {
+		var records = _getDao().selectData(
+				selectFields = [ "audit_log.id", "audit_log.type", "audit_log.datecreated", "audit_log.action", "security_user.email_address", "security_user.known_as"  ]
+				, orderby    = "audit_log.datecreated desc"
+				, startRow   = arguments.startRow
+				, maxRows    = arguments.maxRows
+			);
+
+		return records;
+	}
+
+	/**
+	 * Returns the auditTrail record matching the given ID
+	 *
+	 * @autodoc
+	 * @auditTrailId.hint The ID of the auditTrail you wish to get
+	 *
+	 */
+	public query function getAuditTrail( required string auditTrailId ) {
```

Comment:
I think a better name for this is `getAuditLog()` - "trail" suggests that you could get multiple logs. (also rename `auditTrailId` to `auditLogId`.